specs/RasterizationScale.md

```diff
@@ -0,0 +1,154 @@
+# Background
+The WebView attempts to detects monitor scale changes and applies the monitor DPI scale to all UI produced by the WebView.
+When the host app has a DPI awareness of PerMonitorV2, then all child HWNDs receive WM_DPICHANGED.
+The WebView can check the current monitor DPI scale when it receives WM_DPICHANGED.
+But if the host app is just PerMonitor aware (not v2), then only the top level window receives WM_DPICHANGED.
+The WebView could try to use other metrics (like resize) to check the DPI, but resize is not guaranteed in that scenario.
+From [GitHub feedback](https://github.com/MicrosoftEdge/WebView2Feedback/issues/65),
+we also know that there are consumers of WebView that want to explicitly set a custom DPI scale for WebView.
+
+In this document we describe the updated API. We'd appreciate your feedback.
+
+
+# Description
+To allow the app to control the DPI scale of WebView, we are adding a `RasterizationScale` property to the CoreWebView2Controller interface.
+The RasterizationScale property controls the DPI scaling for all UI in WebView.
+
+In order to maintain compatibility with apps developed before this API existed,
+WebView2 continues to detect monitor scale changes be default and will update the RasterizationScale property.
+When the RasterizationScale property is updated, the `RasterizationScaleChanged` event is raised.
+The app can tell WebView2 to stop updating the RasterizationScale property by changing
+`ShouldDetectMonitorDpiScaleChanges` from the default value of true, to false.
+
+# Examples
+The following code snippets show how to use the RasterizationScale and ShouldDetectMonitorScaleChanges properties,
+and now to listen to the RasterizationScaleChanged event.
+## Win32 C++
+```cpp
+void ViewComponent::SetRasterizationScale(float additionalScale)
+{
+    CHECK_FAILURE(m_controller->put_ShouldDetectMonitorScaleChanges(FALSE));
```

Comment:
Do we need to set shoulddetectmonitorscalechanges to true in this c++ sample?

Reply:
This bit of code should be setting it to false because the app is taking control of setting the RasterizationScale property. The default value is TRUE. I don't expect apps to set it to true, unless for some reason they decided they no longer want to control the rasterization scale and want WebView to return to the default behavior.

Reply:
Not sure what I meant by this comment.